### PR TITLE
Feature/v45 with develop

### PR DESCRIPTION
**Pull request by Jedi2002**
THe PR provides an upgrade handler for the release version 0.14.0

### CODE REVIEW

File: scripts/run.sh [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 killall sifnoded
 
-sifnoded start --trace
+sifnoded start --trace --home ~/.sifnoded/ --p2p.laddr 0.0.0.0:27655  --grpc.address 0.0.0.0:9096 --grpc-web.address 0.0.0.0:9093 --address tcp://0.0.0.0:27659 --rpc.laddr tcp://127.0.0.1:26657
```
Comment (Jedi2002) [inline scripts/run.sh:5]: Added explicit parameters to manage the script better, and use it while running multiple networks while testing 
Default ports are used for the ports used for broadcasting in our test scripts.

File: cmd/sifnoded/cmd/root.go [hunk 2]
```diff
@@ -49,7 +49,6 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 		Use:   "sifnoded",
 		Short: "app Daemon (server)",
 		PersistentPreRunE: func(cmd *cobra.Command, _ []string) error {
-			initClientCtx = client.ReadHomeFlag(initClientCtx, cmd)
```
Comment (canercandan) [inline cmd/sifnoded/cmd/root.go:52]: @Jedi2002 any reason why we are getting rid of this one?

Comment (Jedi2002) [reply cmd/sifnoded/cmd/root.go:52]: The ReadHomeFlag was removed by the SDK in this version. 
We are already setting the home directory to the default when initializing `initClientCtx` so this should not be a problem

**Review by canercandan — APPROVED** [review]
LGTM

**Pull request closed without merging**